packages/kit/src/runtime/client/router.js

```diff
@@ -143,9 +152,8 @@ export class Router {
 			if (svg ? /** @type {SVGAElement} */ (a).target.baseVal : a.target) return;
 
 
-      // Include baseURI if link is inside an svg
-      const base = svg ? document.baseURI : undefined
-			const url = new URL(href, base);
+      const full_href = get_href(/** @type {HTMLAnchorElement | SVGAElement} */ (a));
+			const url = full_href;
```

Comment:
instead of creating two constants that are equal to each other you could combine these lines:
```
      const url = get_href(/** @type {HTMLAnchorElement | SVGAElement} */ (a));
```